mmseg/datasets/custom.py

```diff
@@ -160,6 +187,8 @@ def get_ann_info(self, idx):
     def pre_pipeline(self, results):
         """Prepare results dict for pipeline."""
         results['seg_fields'] = []
+        if self.custom_classes:
+            results['old_id_to_new_id'] = self.old_id_to_new_id
```

Comment:
Could we use something like `label_map` and add some comments about it?

Reply:
I don't understand what you mean by `label_map`. Do you mean changing the key name?

Reply:
Sorry for the ambiguity. 
Yes, `'old_id_to_new_id'` is a bit too long. But `'label_map'` may not be straight forward enough. Any idea?

Reply:
Maybe `class_transform`, `class_id_transform` or `label_transform`?

Reply:
Or `label_map` could be good with a comment explaining it.

Reply:
Then `label_map` it is. We may add comments about key and value of this dict.

Reply:
We may also add comments about what's the purpose of adding this dict.

Reply:
So shall we rename it to `label_map`?